config/samples/sfptpd/default-ocp/kustomization.yaml

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- namespace.yaml
+- ../base
+
+components:
+- ../conf/ptp1
+- ../chronyd/machineconfig-4.8.0
+
+# Corresponds to namespace.yaml
+namespace: sfptpd
+
+# Latest stable
+images:
+- name: onload/sfptpd
+  newName: docker.io/onload/sfptpd
+  newTag: 3.8.0.1000
```

Comment:
I suggest we point to a release candidate on the stable `v3_7` branch after backporting the workflows as `3.8` is unstable.